team/482451/montyhall.py

```diff
@@ -0,0 +1,30 @@
+from random import randint
+
+c = 0
+list2 = []
+
+while c != 100000:
+
+    p = randint(0, 2)
+    list = [0, 0, 0]
```

Comment:
You don't want to name your variables the same as the types that we'll be working with.

Reply:
Totally agree (unfortunately, many programmers break the rule for the function `id`).